mindsdb/integrations/handlers/darts_handler/darts_handler.py

```diff
@@ -1,71 +1,86 @@
 import pandas as pd
 from typing import Optional, Dict
 from darts import TimeSeries
-from darts.models import ARIMA
+from darts.models import AutoARIMA
 from mindsdb.integrations.libs.base import BaseMLEngine
-from mindsdb.integrations.libs.response import HandlerResponse
 
 class DartsModel(BaseMLEngine):
-    
+
     name = "dartsmodel"
 
-    def _init_(self, model_storage, engine_storage) -> None:
-        super()._init_(model_storage, engine_storage)
+    def __init__(self, model_storage, engine_storage) -> None:
+        super().__init__(model_storage, engine_storage)
 
     def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[Dict] = None) -> None:
         """
-        Train an ARIMA model and save it for later usage.
+        Train an AutoARIMA model and save it for later usage.
 
         Args:
             target (str): Name of the target column.
             df (pd.DataFrame): Input DataFrame containing the time series data.
             args (Dict): Additional arguments (if needed).
         """
-        # Ensure df contains a datetime index
-        df.set_index('timestamp', inplace=True)
+
+        # Find the timestamp column in the DataFrame
+        timestamp_column = None
```

Comment:
We recommend getting the `order_by` column directly from the parsed information about the model creation query, instead of this approach. Please check other time series handlers like StatsForecast or NeuralForecast for references on how to achieve that.

Reply:
Hey @paxcema  okay  so what if i do this instead  take another parameter args 
and  
` if args is None or "order_by" not in args:
            # If "order_by" is not provided in args, add a default "order_by" column
            default_order_by_column = '__order_by'
            df[default_order_by_column] = df.index  # Use the DataFrame's index as the default order_by column
            order_by_column = default_order_by_column
        else:
            order_by_column = args["order_by"]  # Extract the 'order_by' column from args

        # Set the order_by column as the index
        df.set_index(order_by_column, inplace=True)`

Reply:
No, there's a better way. Because this is a forecasting engine, we know there _must_ be an `ORDER BY` column passed by the user. Check out this example, you need something almost exactly [[like this](https://github.com/mindsdb/mindsdb/blob/393b524bffbf290ea180298e82184dac8a446bc9/mindsdb/integrations/handlers/statsforecast_handler/statsforecast_handler.py#L93)], feel free to copy and adjust accordingly.